api/README.rst

```diff
@@ -35,44 +35,66 @@ REST Endpoint::
   -> GET /result/<benchmark-group>?[&from=<isotime>&to=<isotime>]
   [{...}, {...}, ...] # Crate data format
 
-The API call will result in this query::
-
-  SELECT benchmark_group as "group",
-         method,
-         build_timestamp as "timestamp",
-         build_version as "version",
-         benchmark_values['round_avg'] as "avg",
-         benchmark_values['round_stddev'] as "stddev"
-  FROM benchmark.history
-  WHERE benchmark_group = ?
-    AND build_timestamp >= ?
-    AND build_timestamp <= ?
-  ORDER BY build_timestamp, method
+The API call will result in this query:
+
+.. code-block:: sql
+
+  SELECT ? as "group",
+    version_info['number'] as "version",
+    version_info['timestamp'] as "version",
+    runtime_stats['min'] as "min",
+    runtime_stats['median'] as "median",
+    runtime_stats['max'] as "max",
+    runtime_stats['stdev'] as "stdev",
+    runtime_stats['variance'] as "variance",
+    statement
+  FROM "benchmark"."history"
+  WHERE statement = ANY(?)
+    AND version_info['build_timestamp'] >= ?
+    AND version_info['build_timestamp'] <= ?
+  ORDER BY version, statement
 
 Table Schema
 ============
 
-::
+The table schema is defined by the `cr8`_ tool.
+
+.. code-block:: sql
 
   CREATE TABLE IF NOT EXISTS "benchmark"."history" (
```

Comment:
there are 2 minor changes to the schema from `cr8`:
* table name
* version_info['timestamp']